### PR TITLE
Doc: Correct several typos.

### PR DESCRIPTION
**Pull request by xylinkun**
Doc: Correct several typos.

### CODE REVIEW

**Comment by icearith** [comment]
This PR should be rebased to squeezed the commits history, as we don't need 6 commits to fixes these typos.

**Comment by wsdjeg** [comment]
we can use squash feature of github. if contributer can not do it.

File: docs/cn/conventions.md [hunk 1]
```diff
@@ -38,7 +38,7 @@ lang: cn
 
 <!-- vim-markdown-toc -->
 
-## Commit emoji 规范
+## Commit 规范
```
Comment (wsdjeg) [inline docs/cn/conventions.md]: emoji should not be deleted.

File: docs/cn/development.md [hunk 2]
```diff
@@ -175,7 +175,7 @@ Further paragraphs come after blank lines.
 
 
 Layer with no associated configuration will be rejected. For instance a layer with just a package and a hook can be easily replaced by the usage of the variable `g:spacevim_custom_plugins`.
-未关联配置的层将会被拒绝.举个例子一个只有包和钩子的层,很简单的就能被替换为变量 `g:spacevim_custom_plugins`.
+未关联配置的layer将会被拒绝.举个例子一个只有包和钩子的layer,就能被很简单地替换为变量 `g:spacevim_custom_plugins`.
```
Comment (wsdjeg) [inline docs/cn/development.md]:
```
s/ layer/模块/
```

**Comment by wsdjeg** [comment]
@ssfjhh Thanks, I just have merged all the changes.

**Pull request merged**